packages/generator/templates/mutation/update__ModelName__.ts

```diff
@@ -17,8 +17,6 @@ export default async function update__ModelName__(
   {where, data}: Update__ModelName__Input,
   ctx: Record<any, any> = {},
 ) {
-  // Don't allow updating
```

Comment:
Can you keep this comment, but move it inside `if (process.env.parentModel)` ?

Reply:
Done!